catalog/spatial/geocoding/spatial-geocoding-plugin/pom.xml

```diff
@@ -65,7 +72,7 @@
                             ddf.catalog.util.impl
                         </Private-Package>
                         <Embed-Dependency>
-                            commons-lang3
+                            commons-lang3, catalog-core-api-impl, platform-util
```

Comment:
❗️ I don't think we need to embed commons-lang here if we use lang 2.6 instead of lang3.